src/AnnotationDialog.js

```diff
@@ -569,106 +542,38 @@ class AnnotationDialog extends EventEmitter {
-        annotationContainerEl.appendChild(annotationEl);
-
-        // Avatar
-        const avatarEl = document.createElement('div');
-        avatarEl.classList.add(constants.CLASS_PROFILE_IMG_CONTAINER);
-        avatarEl.innerHTML = avatarHtml;
-        annotationEl.appendChild(avatarEl);
-
-        // Creator namate & date
-        const profileContainerEl = document.createElement('div');
-        profileContainerEl.classList.add(constants.CLASS_PROFILE_CONTAINER);
-        annotationEl.appendChild(profileContainerEl);
-
-        const userNameEl = document.createElement('div');
-        userNameEl.classList.add(constants.CLASS_USER_NAME);
-        userNameEl.textContent = userName;
-        profileContainerEl.appendChild(userNameEl);
-
-        const createdEl = document.createElement('div');
-        createdEl.classList.add(constants.CLASS_COMMENT_DATE);
-        createdEl.textContent = created;
-        profileContainerEl.appendChild(createdEl);
-
-        // Comment
-        const commentTextEl = document.createElement('div');
-        commentTextEl.appendChild(textEl);
-        annotationEl.appendChild(commentTextEl);
-
-        // Delete button
-        if (!annotation.permissions.can_delete) {
-            return;
-        }
-
-        const deleteBtn = util.generateBtn(
-            [constants.CLASS_BUTTON_PLAIN, CLASS_BUTTON_DELETE_COMMENT],
-            this.localized.deleteButton,
-            ICON_DELETE,
-            constants.DATA_TYPE_DELETE
-        );
-        annotationEl.appendChild(deleteBtn);
-
-        const deleteConfirmEl = document.createElement('div');
-        deleteConfirmEl.classList.add(CLASS_DELETE_CONFIRMATION);
-        deleteConfirmEl.classList.add(constants.CLASS_HIDDEN);
-        annotationEl.appendChild(deleteConfirmEl);
-
-        const confirmMsgEl = document.createElement('div');
-        confirmMsgEl.classList.add(constants.CLASS_DELETE_CONFIRM_MESSAGE);
-        confirmMsgEl.textContent = this.localized.deleteConfirmation;
-        deleteConfirmEl.appendChild(confirmMsgEl);
-
-        const deleteBtnsEl = document.createElement('div');
-        deleteBtnsEl.classList.add(constants.CLASS_BUTTON_CONTAINER);
-        deleteConfirmEl.appendChild(deleteBtnsEl);
-
-        const cancelDeleteBtn = util.generateBtn(
-            [constants.CLASS_BUTTON, CLASS_CANCEL_DELETE],
-            this.localized.cancelButton,
-            this.localized.cancelButton,
-            constants.DATA_TYPE_CANCEL_DELETE
-        );
-        deleteBtnsEl.appendChild(cancelDeleteBtn);
-
-        const confirmDeleteBtn = util.generateBtn(
-            [constants.CLASS_BUTTON, CLASS_BUTTON_DELETE_CONFIRM, constants.CLASS_BUTTON_PRIMARY],
-            this.localized.deleteButton,
-            this.localized.deleteButton,
-            constants.DATA_TYPE_CONFIRM_DELETE
+        const locale = this.locale.substr(0, this.locale.indexOf('-'));
```

Comment:
Can you clarify why we would drop the country from the locale?

Reply:
react-intl only takes the language as a prop but the proper localized messages get passed down. I tested that this works as expected. This was done in a similar fashion as BRUI, elements, etc.

Reply:
Gotcha. Should it be renamed to `language`, then? It doesn't accurately describe the user's locale anymore.